pkg/riff/commands/riff_test.go

```diff
@@ -33,3 +34,14 @@ func TestRiffCommand(t *testing.T) {
 
 	table.Run(t, commands.NewRiffCommand)
 }
+
+func TestRiffMainSubCommands(t *testing.T) {
+	riffCommand := commands.NewRiffCommand(cli.NewDefaultConfig())
+	commands := riffCommand.Commands()
+
+	expectedCount := 7
```

Comment:
While I understand the intent, I don't think this test should stay as is. The risk is that it becomes a PITA more than anything. Maybe just testing that there exist _some_ subcommands is enough?

Reply:
Testing that the command is actually wired is typically a function of acceptance tests. In this case, we also know the command is wired as a side effect of the docs being generated. If the command wasn't wired, then riff_doctor.md wouldn't exist (or CI would fail)